IFC4x3/Properties/t/TechnicalStandard_2M$Q1oBlHFBhUArnmHLc4W/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TechnicalStandard_2MQ1oBlHFBhUArnmHLc4W" Name="TechnicalStandard" UniqueId="96fda072-2ef4-4f2e-b78a-d71c11566120" PropertyType="p_referencevalue" PrimaryDataType="IfcExternalReference" />
```

Comment:
Change type to `IfcDocumentReference`. `IfcExternalREference` is abstract.

Reply:
Done! Commit : 426664c10f4b7a9f9b1eff4dddaa4ee62a65d0e3